build/template/Zotero.dotm/word/vbaProject.bin/Zotero.bas

```diff
@@ -128,7 +128,7 @@ Sub ZoteroCommand(cmd As String, bringToFront As Boolean)
         End If
     Next
     If ThWnd = 0 Then
-        MsgBox ("Word could not communicate with Zotero. Please ensure Firefox is running and try again. If this problem persists, visit http://www.zotero.org/support/word_processor_plugin_troubleshooting")
+        MsgBox ("Word could not communicate with Zotero. Please ensure Firefox or Zotero Standalone is running and try again. If this problem persists, visit http://www.zotero.org/support/word_processor_plugin_troubleshooting")

```

Comment:
"Firefox" is sort of confusing anyway, and we'd have to change this again when the full-featured Zotero for Firefox goes away, so this should probably just be `Please ensure Zotero is running and try again.`. Let's also change that URL to `https` while we're at it (though it doesn't matter in current browsers) and say "see" instead of "visit".